src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/DependencyInjection/HttpClientKeyedRegistrationTest.Descriptors.cs

```diff
@@ -1,51 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
 using System.Net.Http;
 using Xunit;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public partial class HttpClientKeyedRegistrationTest
     {
-        [Fact]
-        public void AddAsKeyed_EmptyNameHttpClientUpdated() // test for workaround for https://github.com/dotnet/runtime/issues/102654
```

Comment:
Is there no value in keeping these tests?

Reply:
None -- they were testing specifically the workaround logic that was removed.